inst/htmlwidgets/lib/rglClass/mouse.src.js

```diff
@@ -512,7 +512,7 @@
       
       handlers.touchstart = function(ev) {
         var touch = ev.touches[0],
-          mouseEvent = new MouseEvent("mousedown",
+          mouseEvent = new MouseEvent("pointerdown",
```

Comment:
I'll need to read up on this, but this doesn't look quite right.  From the little I've read, "pointer" events replace both "mouse" and "touch" events.  Do we still need to listen for `touchstart` events?

Reply:
You may be right, Duncan. I might have gotten in over my head in my beginning understanding of javascript. From the tutorials I had read, they had suggested for basic mouse and touch movements (i.e. you didn't need to listen for multi-touch events) one could just substitute "pointer" for the "mouse" events and get the benefit of listening for mouse and touch events. I therefore just replaced the mouse code with the pointer command 1:1 and tried it and it seemed to work (i.e. I could use touch or my trackpad mouse on my iPad to move the 3D rglwidget object). There may be a better way.